[2024-09-29T12:36:11.149+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:36:11.157+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-23T00:00:00+00:00 [queued]>
[2024-09-29T12:36:11.160+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-23T00:00:00+00:00 [queued]>
[2024-09-29T12:36:11.160+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:36:11.165+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-23 00:00:00+00:00
[2024-09-29T12:36:11.169+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=19802) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:36:11.170+0000] {standard_task_runner.py:72} INFO - Started process 19803 to run task
[2024-09-29T12:36:11.169+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2024-09-23T00:00:00+00:00', '--job-id', '859', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpt9ecq06b']
[2024-09-29T12:36:11.170+0000] {standard_task_runner.py:105} INFO - Job 859: Subtask postgres_to_s3
[2024-09-29T12:36:11.177+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:36:11.194+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-23T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:36:11.195+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:36:11.233+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v01' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-23T00:00:00+00:00'
[2024-09-29T12:36:11.233+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:36:11.246+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:36:11.252+0000] {dag_with_postgres_hooks.py:29} INFO - Saved orders dat in text file get_rders.txt
[2024-09-29T12:36:11.252+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:36:11.257+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:36:11.258+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, run_id=scheduled__2024-09-23T00:00:00+00:00, execution_date=20240923T000000, start_date=20240929T123611, end_date=20240929T123611
[2024-09-29T12:36:11.266+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:36:11.274+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:36:11.275+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-29T12:41:20.624+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:41:20.646+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-23T00:00:00+00:00 [queued]>
[2024-09-29T12:41:20.673+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-23T00:00:00+00:00 [queued]>
[2024-09-29T12:41:20.676+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:41:20.711+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-23 00:00:00+00:00
[2024-09-29T12:41:20.721+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=21230) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:41:20.727+0000] {standard_task_runner.py:72} INFO - Started process 21238 to run task
[2024-09-29T12:41:20.731+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2024-09-23T00:00:00+00:00', '--job-id', '888', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmplih3v5e3']
[2024-09-29T12:41:20.736+0000] {standard_task_runner.py:105} INFO - Job 888: Subtask postgres_to_s3
[2024-09-29T12:41:20.769+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:41:20.823+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-23T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:41:20.827+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:41:20.924+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v01' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-23T00:00:00+00:00'
[2024-09-29T12:41:20.925+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:41:20.955+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:41:20.986+0000] {dag_with_postgres_hooks.py:29} INFO - Saved orders dat in text file get_rders.txt
[2024-09-29T12:41:20.990+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:41:21.001+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:41:21.006+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, run_id=scheduled__2024-09-23T00:00:00+00:00, execution_date=20240923T000000, start_date=20240929T124120, end_date=20240929T124121
[2024-09-29T12:41:21.047+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:41:21.070+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:41:21.072+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
